app/Http/Controllers/ProjectController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class ProjectController extends Controller
+{
+    //
```

Comment:
Make this controller as resource as well

Reply:
Ok